student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_class'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknow', parent_permission: true)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] As mentioned previously there's a typo in the default value for the `name` parameter in the Student class constructor as well so kindly make sure to fix it.